requirements-test.txt

```diff
@@ -1,22 +1,3 @@
-appdirs
-bw2io
-bw2parameters
-docopt
-eight
-fasteners
-future
-lxml
-numpy
-peewee>=3
-psutil
-pyprind
-pytest
-pytest-cov
-python-coveralls
-requests>=1.1.0
-scipy
-stats_arrays
-unicodecsv
-voluptuous
-whoosh
-wrapt
```

Comment:
HI @BenPortner. Can you explain why you are removing dependencies? In my conception the file `requirements-test` should be self-contained and include all dependencies.

Reply:
I will merge, then fix these small comments in a separate commit